packages/design-system/src/Input/Input.tsx

```diff
@@ -24,67 +28,98 @@ import {
 const Input = forwardRef<HTMLInputElement, InputProps>(
   (props, ref): JSX.Element => {
     const {
-      as = "input",
+      className,
       description,
       endIcon,
+      endIconProps,
```

Comment:
What is the purpose of doing this?

Reply:
For passing down the `onClick` function.

Reply:
Will refactor this later and implement icon button here